### PR TITLE
Clean up from #2076

### PR DESCRIPTION
**Pull request by tamird**
#2076 was merged without approval. This PR contains the parts of #2076 that were agreed on plus some cleanup, and removal of the rest.

### CODE REVIEW

**Comment by vivekmenezes** [comment]
I'm more than happy to make some of these changes. This is not the right way to do this. It is extremely disrespectful. I'm human too.

File: client/txn.go [hunk 1]
```diff
@@ -279,19 +268,28 @@ func (txn *Txn) Run(b *Batch) error {
...
-// Commit executes the operations queued up within a batch and commits the
-// transaction. Explicitly committing a transaction is optional, but more
-// efficient than relying on the implicit commit performed when the transaction
-// function returns without error.
-func (txn *Txn) Commit(b *Batch) error {
+// CommitInBatch executes the operations queued up within a batch and
+// commits the transaction. Explicitly committing a transaction is
+// optional, but more efficient than relying on the implicit commit
+// performed when the transaction function returns without error.
+func (txn *Txn) CommitInBatch(b *Batch) error {
```
Comment (vivekmenezes) [inline client/txn.go]: Can we call this CommitBatch

Comment (tamird) [inline client/txn.go]: We can. Don't you think the current name is more descriptive?

**Comment by vivekmenezes** [comment]
LGTM

**Pull request merged**